app/database/db_build.sql

```diff
@@ -10,7 +10,8 @@ CREATE TABLE nuggets(
   title VARCHAR(50) NOT NULL,
   description TEXT,
   img_url VARCHAR(100),
-  author VARCHAR(30) NOT NULL
+  author VARCHAR(30) NOT NULL,
```

Comment:
Kay, [Hu](https://www.forbes.com/profile/jintao-hu/). I see how it is HU, nobody knows who's Hu so now you want to make it so that NOBODY knows who's who, I see, I see